devops/ci/single_node_deploy/Jenkinsfile_single_node_deploy

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env groovy
+
```

Comment:
Rename this file to:
 devops/ci/single_node_deploy/Jenkinsfile_Destroy

Also, add copyright headers.